xtext/org.icyphy.linguafranca/src/org/icyphy/generator/GeneratorBase.xtend

```diff
@@ -1047,9 +1057,8 @@ abstract class GeneratorBase extends AbstractLinguaFrancaValidator {
         val whichReturn = whichBuilder.start().waitFor()
         if (whichReturn == 0) {
             println("SUCCESS")
-            val builder = new ProcessBuilder(#[cmd] + args)
-            builder.directory(new File(directory))
-            return builder
+
+            return ExecutionEnvironment.NATIVE
```

Comment:
Why the indirection with the `ExecutionEnvironment` enum? Couldn't this function simply return the command to be executed directly, i.e. `#[cmd]` or `#["bash", "--login", "-c", "\"", cmd]`? The enum requires common knowledge of the meaning of its values, which doesn't appear like a convenient design.